### PR TITLE
Add licence docs

### PR DESCRIPTION
**Pull request by izavits**
Connects to: https://github.com/resin-io-playground/node.analytics/issues/6

### CODE REVIEW

File: AGREEMENT [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Vesrion 1.0.0
```
Comment (jviotti) [inline AGREEMENT]: Typo?

File: lib/core/index.js [hunk 2]
```diff
@@ -1,9 +1,12 @@
 var Analytics = require('./analytics');
+var fs = require('fs');
+var path = require('path');
 
 var analytics = new Analytics();
 
 analytics.require = require;
 
 analytics.VERSION = require('../../package.json').version;
+analytics.AGREEMENT = fs.readFileSync(path.join(__dirname, '../../AGREEMENT'), 'utf-8');
```
Comment (jviotti) [inline lib/core/index.js]: Can we inline the agreement here? Doing `fs.readFileSync` will not be possible on the browser :(

Comment (izavits) [reply lib/core/index.js]: hmm... right!

File: lib/core/index.js [hunk 3]
```diff
@@ -1,9 +1,14 @@
 var Analytics = require('./analytics');
-
 var analytics = new Analytics();
 
 analytics.require = require;
 
 analytics.VERSION = require('../../package.json').version;
-
-module.exports = analytics;
+analytics.AGREEMENT = 'Version 1.0.0\nThis is a Licence Agreement (the "Agreement") for software modules owned by resin.io (Resin) ' +
```
Comment (jviotti) [inline lib/core/index.js]: Should we get the version from `package.json`? Otherwise we will forget to update it here in the future.

Comment (izavits) [reply lib/core/index.js]: nice catch!

**Review by jviotti — APPROVED** [review]

**Pull request merged**